affordance_util/include/affordance_util/affordance_util.hpp

```diff
@@ -209,7 +215,38 @@ CcModel compose_cc_model_slist(const RobotDescription &robot_description, const
  */
 Eigen::MatrixXd compose_cc_model_slist(const RobotDescription &robot_description, const ScrewInfo &aff_info,
                                        const VirtualScrewOrder &vir_screw_order = VirtualScrewOrder::XYZ);
-
+/** 
+ * @brief Given a urdf pose transform it to a rotation and translation matrix.
+ * 
+ * @param pose URDF pose for conversion.
+ * 
+ * @return Eigen::Matrix4d matrix representation of pose
+ */
+Eigen::Matrix4d urdfPoseToMatrix(const urdf::Pose &pose);
+/** 
+ * @brief Given a model and link get transform from reference.
```

Comment:
Given a urdf model, desired link name, and reference frame name, returns the homogeneous transformation matrix representing the link pose in the reference frame.